### PR TITLE
Fixed small typos in the Karmada operator

### LINKED ISSUE #3649: Fix Karmada Operator typo
**What would you like to be added**:
Just found a typo when going through the code:
https://github.com/karmada-io/karmada/blob/b8abb446b8963b4e1602365daf82f6066083956d/operator/pkg/apis/operator/v1alpha1/type.go#L69

The `GragscefulEviction` should be `GracefulEviction`.

Note: during fix the typo, we should run `make update` to re-generate and commit the auto-generated files as well.

**Why is this needed**:

### PR DESCRIPTION
**Pull request by sbdtu5498**
**What type of PR is this?**
/kind documentation

**What this PR does**
Fixes some typos in the Karmada operator

**Which issue(s) this PR fixes**:
Fixes #3649 

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: charts/karmada-operator/crds/operator.karmada.io_karmadas.yaml [hunk 1]
```diff
@@ -70,7 +70,7 @@ spec:
                           component. In the future, we will provide a more structured
                           way to configure the component. Once that is done, this
                           field will be discouraged to be used. Incorrect settings
-                          on this feild maybe lead to the corresponding component
+                          on this field maybe lead to the corresponding component
```
Comment (RainbowMango) [inline charts/karmada-operator/crds/operator.karmada.io_karmadas.yaml:73]: this issue has been fixed on the master. You might need to rebase your code against master and force-push again.

Comment (sbdtu5498) [reply charts/karmada-operator/crds/operator.karmada.io_karmadas.yaml:73]: Sure! will do it.

Comment (RainbowMango) [reply charts/karmada-operator/crds/operator.karmada.io_karmadas.yaml:73]: Here are the steps for reference:
- git checkout operator-typo-fix
- git fetch upstream
- git rebase upstream/master
- git push origin -f

Comment (sbdtu5498) [reply charts/karmada-operator/crds/operator.karmada.io_karmadas.yaml]: @RainbowMango thanks a lot for the heads up! Now it is all good.

File: operator/pkg/apis/operator/v1alpha1/type.go [hunk 2]
```diff
@@ -66,7 +66,7 @@ type KarmadaSpec struct {
 
 	// FeatureGates enabled by the user.
 	// - Failover: https://karmada.io/docs/userguide/failover/#failover
-	// - GragscefulEviction: https://karmada.io/docs/userguide/failover/#graceful-eviction-feature
+	// - GracefulEviction: https://karmada.io/docs/userguide/failover/#graceful-eviction-feature
```
Comment (yike21) [inline operator/pkg/apis/operator/v1alpha1/type.go:69]: Hi! There is a similar [typo](https://github.com/karmada-io/karmada/blob/master/operator/config/crds/operator.karmada.io_karmadas.yaml#L1460) to fix.

Comment (RainbowMango) [reply operator/pkg/apis/operator/v1alpha1/type.go:69]: That's the auto-generated file that is already included in this PR.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

Thanks @sbdtu5498 and welcome to Karmada community~

**Comment by sbdtu5498** [comment]
> Thanks @sbdtu5498 and welcome to the Karmada community~

Thanks a lot, @RainbowMango! Looking forward to getting more involved.

**Comment by RainbowMango** [comment]
Thanks in advance. We usually publish tasks by GitHub issue(labeled with /help or /good-first-issue), you can take the tasks there.
In addition, we have some proposals as well you can join the review process in the meantime. There will be huge tasks after the proposal is approved.

By the way, why are you interested in Karmada, are you evaluating it? like for your company.

**Comment by sbdtu5498** [comment]
> Thanks in advance. We usually publish tasks by GitHub issue(labeled with /help or /good-first-issue), you can take the tasks there. In addition, we have some proposals as well you can join the review process in the meantime. There will be huge tasks after the proposal is approved.

Sure, I will make sure to tune in next time. Let me know the reviews that are in progress or share the docs maybe on slack (Sanskar Bhushan), I am in Karmada Slack or here. Anything would work. 

> By the way, why are you interested in Karmada, are you evaluating it? like for your company.

I have been using Karmada as a reference to understand Multicluster working and how things are going in sig multicluster. So thought of doing some contribution as well for the good.

**Comment by sbdtu5498** [comment]
Su

> For now, the following proposals are under review:
> 
> * [feat: Add proposal for CronFederatedHPA #3664](https://github.com/karmada-io/karmada/pull/3664)
> * [add proposal for MultiClusterService  #3636](https://github.com/karmada-io/karmada/pull/3636)
> * [Add configurable cluster failover proposal #3634](https://github.com/karmada-io/karmada/pull/3634)

Sure thing! I will start reviewing them in my free time and get more used to Karmada.

**Pull request merged**